Makefile.am

```diff
@@ -217,6 +217,7 @@ if PTRACE_COMMON
 MOSH_CORE_SRCS += src/posix/ptrace/ptrace_common.c
 endif
 
+# We manually renamed misc.c to gc_misc.c because automake gets confused when we do make dist.
```

Comment:
My patch has an alternative workaround(I'll remove this from my #31 atm): https://github.com/higepon/mosh/pull/31/commits/0f888f5b08068feeabee5a82426e6d85ec1ba9ed